src/features/ci-cd/hooks/track-latest-release.ts

```diff
@@ -107,24 +121,22 @@ const updateLatestRelease = async (
 hooks.addPureHook('PATCH', 'resin', 'release', {
 	POSTRUN: async (args) => {
 		const { request } = args;
-		// If we're updating a build by id and setting it successful then we update the application to this build
-		if (request.odataQuery != null) {
-			const keyBind = request.odataQuery.key;
-			// TODO: Support named keys
-			if (keyBind != null && 'bind' in keyBind) {
-				const id = sbvrUtils.resolveOdataBind(request.odataBinds, keyBind);
-				await updateLatestRelease(id, args);
-			}
+		// If we're updating a release by id and setting it successful then we update the application to this build
+		if (
+			request.odataQuery != null &&
```

Comment:
There's no need to check `request.odataQuery` since we no longer use it to short circuit
```suggestion
```